app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/ChatFeedRecyclerAdapter.java

```diff
@@ -483,18 +483,17 @@ public boolean onLongClick(View view) {
     private void handleItemEvents(final WebSearchHolder websearchholder, final int position)  {
         websearchholder.backgroundLayout.setBackgroundColor(ContextCompat.getColor(currContext, isSelected(position) ? R.color.translucent_blue : android.R.color.transparent));
 
-        webquery = MainActivity.webSearch;
-
         final ChatMessage model = getData().get(position);
+        int checkIndex = model.getContent().lastIndexOf("#");
         websearchholder.messageStar.setVisibility( (model.isImportant()) ? View.VISIBLE : View.GONE);
-        websearchholder.text.setText(model.getContent());
+        websearchholder.text.setText(model.getContent().substring(0,checkIndex));
```

Comment:
if you query something like "search for dog", the server responses with 
answer : Here is a search result
websearch : dog
What you are doing is you are converting `answer` to `answer+"#"+websearch`
 So in the above case answer will become `Here is a search result#dog` 
Then you find index of "#" using `int checkIndex = model.getContent().lastIndexOf("#");` and separate them again.
Now suppose I query "search for abc#abc"
answer : Here is a search result
websearch : abc#abc
Now when you combine them answer will become `Here is a search result#abc#abc`.
Now when you use `int checkIndex = model.getContent().lastIndexOf("#");` and separate them, you'll get answer as `Here is a search result#abc` and websearch as `abc`.
</br>
Though I am not getting expected result "search for abc#abc". The server is interpreting '#' as space and ignoring it. Need to confirm it from @Orbiter